.github/workflows/editorconfig-checks.yaml

```diff
@@ -0,0 +1,16 @@
+name: Editorconfig checks
+on:
+  push:
+  pull_request:
+
+jobs:
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Editorconfig checks
```

Comment:
This should be an action, e.g. “Check files for EditorConfig compliance”

Reply:
Fixed.